app/views/registration_wizard/ineligible_for_funding/early_years/_outside_catchment_or_not_on_early_years_register.html.erb

```diff
@@ -1,14 +1,9 @@
 <% content_for :title do %>
-  <%= @form.errors.present? ? "Error: " : nil %> DfE scholarship funding is not available
+  <%= @form.errors.present? ? "Error: " : nil %> Funding eligibility
 <% end %>
 
-<h1 class="govuk-heading-xl">DfE scholarship funding is not available</h1>
+<h1 class="govuk-heading-xl">Funding eligibility</h1>
 
-<p class="govuk-body">To be eligible for scholarship funding for <%= localise_sentence_embedded_course_name(course) %> you must:</p>
+<p class="govuk-body">You're not elgible for scholarship funding for the <%= localise_sentence_embedded_course_name(course) %> because you or your employer is not registered on the Ofsted Early Years Register or with a registered Childminder Agency.</p>
```

Comment:
```suggestion
<p class="govuk-body">You're not eligible for scholarship funding for the <%= localise_sentence_embedded_course_name(course) %> because you or your employer is not registered on the Ofsted Early Years Register or with a registered Childminder Agency.</p>
```